elastic/datadog_checks/elastic/elastic.py

```diff
@@ -496,6 +514,29 @@ def _get_es_version(self, config):
         self.log.debug("Elasticsearch version is %s" % version)
         return version
 
+    def _get_index_metrics(self, config):
+        cat_url = '/_cat/indices?format=json&bytes=b'
+        index_url = urlparse.urljoin(config.url, cat_url)
+        index_resp = self._get_data(index_url, config)
+        index_stats_metrics = self.INDEX_STATS_METRICS
+        for idx in index_resp:
+            tags = config.tags + ['index_name:' + idx['index']]
+            index_data = {
+                'docs_count':         idx['docs.count'],
+                'docs_deleted':       idx['docs.deleted'],
+                'primary_shards':     idx['pri'],
+                'replica_shards':     idx['rep'],
+                'primary_store_size': idx['pri.store.size'],
+                'store_size':         idx['store.size'],
+                'health':             idx['health'],
```

Comment:
Do we have to convert this to a value, or is this returned as a number? I'm seeing the docs mention it returns a string like yellow or green, but could be wrong here 🙂 https://www.elastic.co/guide/en/elasticsearch/reference/6.2/cat-indices.html

Reply:
I can convert to the numbers, should `green`=0, 'yellow' =1, and `red`=2 like for service check statuses?

Reply:
Yeah I think thats the best option.